lib/static/modules/actions.js

```diff
@@ -270,10 +270,15 @@ function filterFailedBrowsers(suites = []) {
 }
 
 function filterAcceptableBrowsers(suites = []) {
-    return filterBrowsers(suites, [isAcceptable, filterOpenedStateResult]);
+    return filterBrowsers(suites, [isAcceptable, filterOpenedStateResult], getOpenedBrowserResult);
 }
 
-function getOpenedBrowserResult({result, retries, state}) {
+function getBrowserResult(bro) {
+    return bro.result;
+}
+
+function getOpenedBrowserResult(bro) {
+    const {result, retries, state} = bro;
```

Comment:
I moved destructuring into the function purposely, so that anyone could see that input variable is `browser`.